cmd/ctr/images.go

```diff
@@ -58,7 +58,7 @@ var imagesListCommand = cli.Command{
 		tw := tabwriter.NewWriter(os.Stdout, 1, 8, 1, ' ', 0)
 		fmt.Fprintln(tw, "REF\tTYPE\tDIGEST\tSIZE\tPLATFORM\tLABELS\t")
 		for _, image := range imageList {
-			size, err := image.Size(ctx, cs)
+			size, err := image.Size(ctx, cs, platforms.Default())
```

Comment:
Shouldn't this `SIZE` field contain non-default platform blobs?

Reply:
I think that might be confusing (although "size" for manifest-list based images is already a potentially confusing concept). However, given the on-disk size will only contain the default platform, it makes sense to me that the reported size would only contain that set of image layers.

If `SIZE` reflected the entire set of a large group of platform images (especially multi-platform images containing Windows layers) that could potentially report a very large number that wouldn't reflect the actual image use on a particular platform. I know that my recent `mplatform/mquery` image is > 300MB on Windows, but < 3MB on Linux, for example.